action.yaml

```diff
@@ -18,5 +18,17 @@ inputs:
     required: false
     default: ""
 runs:
-  using: 'node20'
-  main: 'dist/index.mjs'
+  using: "composite"
+  steps:
+    - name: Setup Bun
+      uses: oven-sh/setup-bun@v1
+      with:
+        bun-version: 1.x.x
+
+    - name: Install dependencies
+      run: bun i
+      shell: bash
```

Comment:
I don't think this is correct. This will install the target project's dependencies with Bun.

Reply:
oh makes sense!